pkg/VSConfigFinder/VSConfigFinder.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
```

Comment:
Do we have to create a nupkg to create a choco package? What do we need to produce to publish to WinGet?

Reply:
For WinGet, we use what's published on GitHub. Nuspec is required to create a choco package (https://docs.chocolatey.org/en-us/create/create-packages#okay-how-do-i-create-packages).

Reply:
Taking a look, choco expects a nupkg so this should be good.